docker/README.md

```diff
@@ -7,7 +7,7 @@ build will error out complaining of missing files.
 
 ### Build the container locally
 Decide where you would like the built container to be stored.  The 
```

Comment:
If this is an inactive `Pull Request`, may I have your permission to close it or update?
 😄 
Thanks!

Reply:
@levonet

Reply:
Yes. You can do it on your own.